.github/workflows/ci.yml

```diff
@@ -117,13 +103,25 @@ jobs:
           projectId: ${{ secrets.ATLAS_QA_PROJECT_ID}}
           apiKey: ${{ secrets.ATLAS_QA_PUBLIC_API_KEY}}
           privateApiKey: ${{ secrets.ATLAS_QA_PRIVATE_API_KEY }}
-          differentiator: flutter-android
-
-# Linux jobs
+          differentiator: ${{ matrix.package }}-${{ matrix.platform }}
 
-  build-linux:
-    runs-on: ubuntu-latest
-    name: Build Linux
+  build-native:
+    runs-on: ${{ matrix.runner }}-latest
+    name: Build native
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { preset: windows-x64, runner: windows }
```

Comment:
This will produce an artifact called `librealm-windows-x64`. This is problematic because the release workflow expect the artifact names to match the target platform name (i.e. we need to upload the binary as `windows`, not `windows-x64`).